Libs/MRML/Core/vtkMRMLPlotNode.h

```diff
@@ -0,0 +1,249 @@
+/*==============================================================================
+
+  Copyright (c) Kapteyn Astronomical Institute
+  University of Groningen, Groningen, Netherlands. All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  This file was originally developed by Davide Punzo, Kapteyn Astronomical Institute,
+  and was supported through the European Research Consil grant nr. 291531.
+
+==============================================================================*/
+
+#ifndef __vtkMRMLPlotNode_h
+#define __vtkMRMLPlotNode_h
+
+#include <string>
+#include <vector>
+
+// MRML Includes
+#include "vtkMRMLNode.h"
+
+class vtkMRMLStorageNode;
+class vtkMRMLTableNode;
+
+// VTK Includes
+class vtkColor4ub;
+class vtkPlot;
+class vtkTable;
+
+/// \brief MRML node to represent a Plot object
```

Comment:
No. a vtkMRMLPlotNode is only one vtkPlot which it can handle only two coloumns (x and y) from one Table. So if you want to plot two tables, you need to create two vtkMRMLPlotNode referencing different tables and then you need to save their IDs in a vtkMRMLPlotLayoutNode. The view node will be able to overlay the two  vtkMRMLPlotNode.